src/main/java/racingcar/RacingCar.java

```diff
@@ -0,0 +1,208 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Iterator;
+import java.util.Vector;
+
+/**
+ * RacingCar 클래스, <br>
+ * 유저에게 자동차 이름과 경주를 시도할 횟수를 입력 받음 <br>
+ * 입력된 횟수만큼 0~9 사이의 랜덤 정수 값을 발생시켜 4 이상일 경우 전진하는 것으로 저장 <br>
+ * 각 참가자들이 전진한 횟수를 비교하여 가장 큰 값을 가진 우승자를 찾음 <br>
+ * 우승자 출력
+ */
+public class RacingCar {
+    private Vector<User> userList;
```

Comment:
```suggestion
    private List<User> userList;
```
인터페이스로 선언해도 좋을 것 같습니다!